test/unify_package_rename_suggestors/unify_rename_suggestor_test.dart

```diff
@@ -0,0 +1,427 @@
+      unify.AutocompleteChangeDetails();
+      random_rmui_namespace.BackdropObject;
+      BadgeOriginHorizontal;
+      MenuPopoverOrigin();
+      (Slider()..marks = [SliderMark(value: 10)])();
+    }
+''',
+        );
+      });
+
+      test('ButtonColor updates', () async {
+        await testSuggestor(
+          input: /*language=dart*/ '''
+    import 'package:react_material_ui/react_material_ui.dart' as mui;
+    import 'package:react_material_ui/react_material_ui.dart';
+    
+    content() {
+      mui.ButtonColor.success;
+      mui.ButtonColor.wsdBtnInverse;
+      mui.ButtonColor.wsdBtnLight;
+      mui.ButtonColor.wsdBtnWhite;
+      ButtonColor.success;
+      ButtonColor.wsdBtnInverse;
+      ButtonColor.wsdBtnLight;
+      ButtonColor.wsdBtnWhite;
+    }
+''',
+          expectedOutput: /*language=dart*/ '''
+    import 'package:react_material_ui/react_material_ui.dart' as mui;
+    import 'package:react_material_ui/react_material_ui.dart';
+    
+    content() {
+      unify.ButtonColor.success;
+      unify_wsd.WsdButtonColor.wsdBtnInverse;
+      unify_wsd.WsdButtonColor.wsdBtnLight;
+      unify_wsd.WsdButtonColor.wsdBtnWhite;
+      ButtonColor.success;
+      unify_wsd.WsdButtonColor.wsdBtnInverse;
+      unify_wsd.WsdButtonColor.wsdBtnLight;
+      unify_wsd.WsdButtonColor.wsdBtnWhite;
+    }
+''',
+        );
+      });
+
+      test('except when they are not from react_material_ui', () async {
+        await testSuggestor(
+          input: /*language=dart*/ '''
+    import 'package:over_react/over_react.dart';
+    
+    // Shadows the RMUI factories
+    UiFactory Alert;
+    UiFactory LinkButton;
+    UiFactory MuiList;
+    UiFactory WorkivaMuiThemeProvider;
+    class BackdropTimeoutObject {}
+    abstract class ButtonColor {
+      static const String wsdBtnLight = 'wsdBtnLight';
+    }
+    
+    content() {
+      Alert()();
+      LinkButton()();
+      MuiList()();
+      WorkivaMuiThemeProvider()();
+      BackdropTimeoutObject;
+      ButtonColor.wsdBtnLight;
+    }
+''',
+        );
+      });
+    });
+
+    group('fixme comments', () {
+      test('for specific components that need manual intervention', () async {
+        await testSuggestor(
+          input: /*language=dart*/ '''
+    import 'package:react_material_ui/react_material_ui.dart' as mui;
+    import 'package:react_material_ui/react_material_ui.dart';
+    
+    content() {
+      mui.Badge()();
+      Badge()();
+      mui.LinearProgress()();
+      LinearProgress()();
+    }
+''',
+          expectedOutput: /*language=dart*/ '''
+    import 'package:react_material_ui/react_material_ui.dart' as mui;
+    import 'package:react_material_ui/react_material_ui.dart';
+
+    content() {
+      // FIXME(unify_package_rename) Check what theme provider is wrapping this component: if it is a UnifyThemeProvider, remove this FIXME - no action is required; otherwise, migrate this component back to Web Skin Dart.
```

Comment:
#nit it looks like this fixme insertion isn't idempotent, but the idempotency tests are passing because on the second run, the `unify.` references are unresolved because the imports hadn't been updated.

I don't think this is worth fixing since it's so uncommon, but I thought I'd call it out.

Reply:
Oh that's a good catch! Okay I'll keep that in mind and try not to re-run the codemod too much